website/docs/en/config/devtool.mdx

```diff
@@ -4,54 +4,93 @@ import WebpackLicense from '@components/WebpackLicense';
 
 # Devtool
 
-The `devtool` configuration is used to control the behavior of the Source Map generation.
+Choose a style of source mapping to enhance the debugging process. These values can affect build and rebuild speed dramatically.
+
+Use the [SourceMapDevToolPlugin](/plugins/webpack/source-map-dev-tool-plugin) or [EvalSourceMapDevToolPlugin](/plugins/webpack/eval-source-map-dev-tool-plugin) for a more fine grained configuration.
 
 - **Type:**
 
 ```ts
-type Devtool =
-  | false
-  | 'eval'
-  | 'cheap-source-map'
-  | 'cheap-module-source-map'
-  | 'source-map'
-  | 'inline-cheap-source-map'
-  | 'inline-cheap-module-source-map'
-  | 'inline-source-map'
-  | 'inline-nosources-cheap-source-map'
-  | 'inline-nosources-cheap-module-source-map'
-  | 'inline-nosources-source-map'
-  | 'nosources-cheap-source-map'
-  | 'nosources-cheap-module-source-map'
-  | 'nosources-source-map'
-  | 'hidden-nosources-cheap-source-map'
-  | 'hidden-nosources-cheap-module-source-map'
-  | 'hidden-nosources-source-map'
-  | 'hidden-cheap-source-map'
-  | 'hidden-cheap-module-source-map'
-  | 'hidden-source-map'
-  | 'eval-cheap-source-map'
-  | 'eval-cheap-module-source-map'
-  | 'eval-source-map'
-  | 'eval-nosources-cheap-source-map'
-  | 'eval-nosources-cheap-module-source-map'
-  | 'eval-nosources-source-map';
+type Devtool = 'string' | false;
 ```
 
 - **Default:** `eval`
```

Comment:
This default value is not correct. Can we clarify the default value in different mode? And Rspack CLI also has a different default value.